scripts/bastion/sync-to-bastion.sh

```
@@ -0,0 +1,127 @@
+# over SSH, WITHOUT changing the default, by setting:
+#   BASTION_SSH_HOST   explicit host to ssh/scp to (raw ssh, no gcloud)
+#   BASTION_SSH_USER   login user for that host (defaults to the local user)
+upload_archive() { :; }
+remote_exec() { :; }
+if [ -n "${BASTION_SSH_HOST:-}" ]; then
+  SSH_HOST="${BASTION_SSH_HOST}"
+  SSH_USER="${BASTION_SSH_USER:-$(id -un)}"
+  SSH_TARGET="${SSH_USER}@${SSH_HOST}"
+  echo "==> transport: direct ssh to ${SSH_TARGET}"
+  upload_archive() { scp "${ARCHIVE}" "${SSH_TARGET}:${REMOTE_ARCHIVE}"; }
+  remote_exec() { ssh "${SSH_TARGET}" "$1"; }
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)